insights/specs/__init__.py

```diff
@@ -599,6 +600,7 @@ class Specs(SpecSet):
     udev_persistent_net_rules = RegistryPoint()
     ulimit_hard = RegistryPoint()
     uname = RegistryPoint()
+    uname_r = RegistryPoint()
```

Comment:
The existing `uname` spec covers the `uname_r`.

Reply:
Removed in latest patch